model/src/main/proto/spine_examples/chatspn/chat/chat.proto

```diff
@@ -45,7 +45,7 @@ message Chat {
     ChatId id = 1;
 
     // List of the chat members.
-    repeated spine.core.UserId member = 2 [(required) = true, (distinct) = true];
+    repeated spine.core.UserId member = 2 [(distinct) = true];
```

Comment:
I think if the member's list can be empty It should be described in the documentation. Please review.